packages/studiocms_core/src/strings.ts

```diff
@@ -9,9 +9,9 @@ export const CoreStrings = {
 };
 
 export const DbErrors = {
-	AstroConfigOutput: "Astro Studio CMS is only supported in 'Output: server' SSR mode.",
+	AstroConfigOutput: "Astro StudioCMS is only supported in 'Output: server' SSR mode.",
```

Comment:
Old Name

Reply:
fixed as of [aae4d48](https://github.com/astrolicious/studiocms/pull/352/commits/aae4d48b71e3d860774b1789626ff4bb1c6c1641)